testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -218,9 +219,9 @@ def test_force_set_fail(self):
 
 class TestAtomGroup(TestCase):
     """Tests of AtomGroup; selections are tested separately."""
-    # all tests are done with the AdK system (PSF and DCD)
-    # sequence: http://www.uniprot.org/uniprot/P69441.fasta
-    # >sp|P69441|KAD_ECOLI Adenylate kinase OS=Escherichia coli (strain K12) GN=adk PE=1 SV=1
+    # all tests are done with the AdK system (PSF and DCD) sequence:
+    # http://www.uniprot.org/uniprot/P69441.fasta >sp|P69441|KAD_ECOLI
```

Comment:
`>sp|P69441|KAD_ECOLI Adenylate kinase OS=Escherichia coli (strain K12) GN=adk PE=1 SV=1` is the header that is read from the fasta file. I am not sure it make sense to split it. I would keep it on a single line, even if that line is longer than 79 characters.